knexfile.js

```diff
@@ -0,0 +1,44 @@
+module.exports = {
+  development: {
+    client: "sqlite3",
+    connection: {
+      filename: "./data/lambda.sqlite3"
+    },
+    useNullAsDefault: true,
+    migrations: {
+      tableName: "knex_migrations"
+    }
+  },
+
+  staging: {
+    client: "postgresql",
+    connection: {
+      database: "my_db",
+      user: "username",
+      password: "password"
+    },
+    pool: {
+      min: 2,
+      max: 10
+    },
+    migrations: {
+      tableName: "knex_migrations"
+    }
+  },
+
+  production: {
```

Comment:
You could remove the stuff that's not development here since we're only using development.